src/main/java/supersymmetry/common/metatileentities/multi/electric/MetaTileEntityFrothFlotationTank.java

```diff
@@ -0,0 +1,73 @@
+package supersymmetry.common.metatileentities.multi.electric;
+
+import gregtech.api.capability.impl.MultiblockRecipeLogic;
+import gregtech.api.metatileentity.MetaTileEntity;
+import gregtech.api.metatileentity.interfaces.IGregTechTileEntity;
+import gregtech.api.metatileentity.multiblock.IMultiblockPart;
+import gregtech.api.metatileentity.multiblock.RecipeMapMultiblockController;
+import gregtech.api.pattern.BlockPattern;
+import gregtech.api.pattern.FactoryBlockPattern;
+import gregtech.api.unification.material.Materials;
+import gregtech.client.renderer.ICubeRenderer;
+import gregtech.client.renderer.texture.Textures;
+import gregtech.client.utils.TooltipHelper;
+import gregtech.common.blocks.BlockBoilerCasing.BoilerCasingType;
+import gregtech.common.blocks.BlockMetalCasing.MetalCasingType;
+import gregtech.common.blocks.BlockTurbineCasing;
+import gregtech.common.blocks.MetaBlocks;
+import gregtech.common.blocks.StoneVariantBlock;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.client.resources.I18n;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.world.World;
+import supersymmetry.api.recipes.SuSyRecipeMaps;
+import supersymmetry.common.blocks.BlockMultiblockTank;
+import supersymmetry.common.blocks.SuSyBlocks;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.List;
+
+public class MetaTileEntityFrothFlotationTank extends RecipeMapMultiblockController {
+    public MetaTileEntityFrothFlotationTank(ResourceLocation metaTileEntityId) {
+        super(metaTileEntityId, SuSyRecipeMaps.FROTH_FLOTATION);
+        this.recipeMapWorkable = new MultiblockRecipeLogic(this, true);
+    }
+
+    public MetaTileEntity createMetaTileEntity(IGregTechTileEntity tileEntity) {
+        return new MetaTileEntityFrothFlotationTank(this.metaTileEntityId);
+    }
+
+    protected BlockPattern createStructurePattern() {
+        return FactoryBlockPattern.start()
+                .aisle("   B   ", "   B   ", "   B   ", "       ", "       ")
+                .aisle("  AAA  ", "  AAA  ", "  AAA  ", "  AAA  ", "  AAA  ")
+                .aisle(" AAAAA ", " ABBBA ", " ABBBA ", " ADDDA ", " A   A ")
+                .aisle("BAAAAAB", "BABBBAB", "BABBBAB", " ADBDA ", " A E A ")
+                .aisle(" AAAAA ", " ABBBA ", " ABBBA ", " ADDDA ", " A   A ")
+                .aisle("  AAA  ", "  AAA  ", "  AAA  ", "  AAA  ", "  AAA  ")
+                .aisle("   B   ", "   B   ", "   S   ", "       ", "       ")
+                .where('S', selfPredicate())
+                .where('A', states(MetaBlocks.METAL_CASING.getState(MetalCasingType.STAINLESS_CLEAN))
+                        .or(autoAbilities(true, true, true, true, true, true, false)))
```

Comment:
You can just do autoAbilities() here - the muffler mechanics are controlled by another method which needs overriding to enable.